src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -138,6 +140,9 @@ export const UserInformationContent: React.FunctionComponent<UserInformationCont
           <UserAvatar user={user} showStatus large />
         </div>
         <div className="UserInformationContent__text-container">
+          <span className="UserInformationContent__status-prefix">
+            Available
+          </span>
```

Comment:
can you explain why was that added, please? not sure about the the need of it.

Reply:
Accidentally added by reverting those github/env changes.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/6db8a73718475d1f8e780610acf3becd6f5e4c3b